runtime/openshift.go

```diff
@@ -45,7 +45,7 @@ func (o openshift) shouldJoinNetworks() (bool, PostSandboxCreate, PostSandboxDes
 	if err != nil {
 		// The plugins could not be defined. ex: when using oc cluster up
 		// or a pure k8s cluster. Therefore making this a notice.
-		log.Infof("unable to retrieve the network plugin, defaulting to not joining networks - %v", err)
+		log.Debugf("unable to retrieve the network plugin, defaulting to not joining networks - %v", err)
```

Comment:
This log statement is stated to be non-critical in the comment above, so figured it was ok to set it to "Debug"

Reply:
I really think we should be checking the error before eating the log.  But then again we're not doing anything with the error `return false, nil, nil`. Feels like we're really ignore a real error.

Reply:
I'm not sure what errors might result here / how we would handle them. I can file an issue if that would make sense.

Reply:
I think this should be of a higher log level than a debug. Maybe Info, probably not Warning. How often do we expect this to occur?

Reply:
It happens during every `apb` provision/deprovision, at least with the 3.9 cluster I'm using.

Reply:
@eriknelson In the CLI tool since we are executing against an `oc cluster up` environment frequently it shows up 100% of the time on INFO.